### DISCUSSION TITLE
use IndexedDB in tauri apps .  and the devtools

### DISCUSSION THREAD

**Discussion by noor-tg** [tauri-apps/tauri#4876] [Q&A] [answered]
hi, 

I want to use IndexedDB in my app . when I checked the devtools `storage` section there was no tab for indexedDB. 

is IndexedDB supported by tauri ? or just the devtools is not viewing it ?

thank you.

**Comment by FabianLars** [comment, marked as answer]
IndexedDB should work just fine on all 3 OSes.
No idea why it's not visible in the devtools on Linux (idk about macos), but on Windows it is displayed fine 🤔

**Reply by noor-tg** [reply]
after adding indexedb db the data showed in the devtools.

so maybe this a bug.

**Reply by FabianLars** [reply]
Looks like this is just how WebKit behaves, nothing we can do here afaik

**Reply by ohellel** [reply]
I can't find a way to change data directory to documents on windows or desktop or any other location.

**Comment by ZeeshanAhmadKhalil** [comment]
Is there a better way to visualise index DB data. Inside devtools it's hard to find a specific row since data is not ordered.
